shared/locales/en/website-partners.json

```diff
@@ -28,279 +28,303 @@
-			"org-description-paragraphs": [
-				[
-					{
-						"text": "The Sierra Leone Association of Ebola Survivors (SLAES) is an organization committed to supporting individuals who have overcome the Ebola virus. SLAES offers comprehensive assistance, including access to medical care, psychological support, and ongoing advocacy to ensure that survivors’ rights and needs are recognized and addressed."
-					}
-				],
-				[
-					{
-						"text": "Through its work, SLAES aims to help survivors rebuild their lives and foster resilience in affected communities."
-					}
-				]
-			],
-			"org-quote": [
+			[
 				{
-					"text": "It’s more than financial support—it’s a step "
-				},
+					"text": "UPBSA’s mission is to provide a safe, supportive space that equips its members with the tools and opportunities to lead dignified lives."
+				}
+			]
+		],
+		"org-quote": [
+			{
+				"text": "Social Income has empowered the most vulnerable to "
+			},
+			{
+				"text": "launch small businesses, ",
+				"color": "accent"
+			},
+			{
+				"text": "generating steady income through its dependable monthly support."
+			}
+		],
+		"org-quote-author": "Paul Osman Kabia",
+		"org-photo": ""
+	},
+	"sierra-leone-association-of-ebola-survivors": {
+		"org-slug": "sierra-leone-association-of-ebola-survivors",
+		"org-short-name": "SLAES",
+		"org-long-name": "Sierra Leone Association of Ebola Survivors",
+		"org-foundation": "2015",
+		"org-image": "slaes.png",
+		"org-country": "SL",
+		"org-headquarter": "Kenema",
+		"org-category": "Local NGO",
+		"org-focus-sdg-numbers": [4, 5],
+		"partnership-start": "2024",
+		"recipients-total": "20",
+		"recipients-active": "17",
+		"recipients-former": "2",
+		"recipients-suspended": "1",
+		"org-mission": "The community-based NGO supports individuals who have survived the Ebola virus, focusing on those affected by the devastating 2014 outbreak in Sierra Leone.",
+		"org-description-paragraphs": [
+			[
 				{
-					"text": "towards dignity and resilience.",
-					"color": "accent"
+					"text": "The Sierra Leone Association of Ebola Survivors (SLAES) is an organization committed to supporting individuals who have overcome the Ebola virus. SLAES offers comprehensive assistance, including access to medical care, psychological support, and ongoing advocacy to ensure that survivors’ rights and needs are recognized and addressed."
 				}
 			],
-			"org-quote-author": "Paul Osman Kabia",
-			"org-photo": ""
-		}
-	]
+			[
+				{
+					"text": "Through its work, SLAES aims to help survivors rebuild their lives and foster resilience in affected communities."
+				}
+			]
+		],
+		"org-fundraiser-text": [
+			{
+				"text": "SLAES is currently fundraising: "
+			},
+			{
+				"text": "Rebuilding Lives",
+				"href": "https://socialincome.org/campaign/MZmXEVHlDjOOFOMk82jW"
+			}
+		],
```

Comment:
I realised that SLAES is only raising a fundraiser in English 😅
Need some translation help here as well